tasmota/support_tasmota.ino

```diff
@@ -544,6 +544,7 @@ void ExecuteCommandPower(uint32_t device, uint32_t state, uint32_t source)
   SetPulseTimer((device -1) % MAX_PULSETIMERS, 0);
 
   static bool interlock_mutex = false;    // Interlock power command pending
+  bool perform_interlock_delay = false;
```

Comment:
Wrong location. As ExecuteCommand is re-entrant it will clear this flag every time disabling the needed delay. This should be moved between the interlock_mutex flags.

Reply:
As the variable is not `static` that shouldn't be an issue, or?

From a stylistic point it would make sense to move it into the scope where it is useful which would be the same place.